### PR TITLE
Feature/about page

### LINKED ISSUE #40: Page: About
- [ ] About the Project
- [ ] About PONI
- [ ] About the Nuclear Policy Working Group
- [ ] Principle Researchers
- [ ] Development Team

### PR DESCRIPTION
**Pull request by aleesteele**
closes #40 

A couple of questions:
- Should the html for the aside be added directly in the forestry page? It will then be styled directly in the sass file. 
- Is the bio the correct length? I saw in the 'About the Project' section that the text block is pretty long. Not sure if you/Tucker wanted it shorter.

Notes for completion:
- The `h2` titles aren't the proper color, will have to fix this, due to an error with the overriding styles.
- The breadcrumb will be added in later (config file)

### CODE REVIEW

File: assets/_sass/base/_typography.scss [hunk 1]
```diff
@@ -60,6 +60,7 @@ h5 {
 
   h3 {
     @extend %post__content-h3;
+    margin-bottom: 1rem;
```
Comment (aleesteele) [inline assets/_sass/base/_typography.scss]: added design s

File: _includes/aside-heading.html [hunk 2]
```diff
@@ -0,0 +1,22 @@
+{% assign siteLink = site.link %}
+
+<div class="aside-heading">
```
Comment (jnschrag) [inline _includes/aside-heading.html]: This should use an `<aside>` element instead of a div.

File: assets/_sass/components/_aside-heading.scss [hunk 3]
```diff
@@ -0,0 +1,41 @@
...
+.aside-heading {
+  display: none;
+  float: right;
+  width: 230px;
+  margin: 0 1em 1em;
+  padding: 1em;
+  background-color: $color__off-white;
+
+  @include breakpoint('medium') {
+    display: inline;
```
Comment (jnschrag) [inline assets/_sass/components/_aside-heading.scss]: This should be `display: block`

File: assets/_sass/components/_aside-heading.scss [hunk 4]
```diff
@@ -0,0 +1,41 @@
+/*===============================
+=         Aside Heading         =
+===============================*/
+
+.aside-heading {
+  display: none;
```
Comment (jnschrag) [inline assets/_sass/components/_aside-heading.scss]: Why is it being hidden on mobile?

**Comment by aleesteele** [comment]
These were written terribly, my sincere apologies. Taking a fresh look at them this afternoon, I can barely recognize the logic...
- Deleted some of the extraneous styles, as I'm not sure if the `box-shadow` should still display for the links (they don't appear on Zeplin). Also if the aside appears normally on mobile, then there's no need to have a breakpoint... unless the aside should be a full width div?
- Fixed the list, so that it has the properly syntax.

**Comment by jnschrag** [comment]
This is much better! Still a few tweaks to make, but to answer your questions first:

- Let's remove the box-shadow from these links
- On mobile, it would be better to remove the float & margins on the side, and make the width 100%. It should only have a set width & be floated on medium screens and up (when there's room to accommodate it).
- Can you add `margin-bottom: 1.5rem` to the list elements?

**Comment by aleesteele** [comment]
- Removed the box shadow on those links
- Removed float & margins on mobile
- Added `margin-bottom: 1.5rem` to the list elements

**Pull request merged**